### PR TITLE
Handle Parametrized Tests

### LINKED ISSUE #1: Broken dependency path
```
INTERNALERROR> FileNotFoundError: [Errno 2] No such file or directory: '/Users/walles/src/prsummary/.pytest-avoidance/python-3.7.3-e1fbe50eb09eddf0d97534bcccab1fc1/systemtest/test_summary_output.py/test_validate_markdown_output[/Users/walles/src/prsummary/systemtest/markdown/38488697.md].deps'
```

### PR DESCRIPTION
**Pull request by walles**
Fixes #1

### CODE REVIEW

File: pytest_avoidance.py [hunk 1]
```diff
@@ -10,9 +10,12 @@
 import errno
 import sys
 import os
+import re
 
 CACHEROOT = None
 
+PARAMETERS_RE = re.compile(r'\[([^]]+)\]')
```
Comment (blueyed) [inline pytest_avoidance.py:17]: This could/should be anchored at the end probably (`$`).

Comment (walles) [reply pytest_avoidance.py:17]: Can you come up with a use case where that would make a difference?

Comment (blueyed) [reply pytest_avoidance.py:17]: In theory you could have a test `test_foo[` that gets parametrized as `test_foo[[param1]`.
It won't make much a difference though I guess.

FWIW: I'm not using pytest-avoidance myself, just gave it a quick spin today.

Comment (walles) [reply pytest_avoidance.py:17]: If you can show an actual test case where this breaks I'll be all ears :).

Here's inspiration:
https://github.com/walles/pytest-avoidance/blob/75fbc9ded38f1d881f4f67eec25390c291247966/tests/avoidance_test.py#L29-L37

**Pull request merged**